pkg/ebpf/tracee.go

```diff
@@ -288,13 +288,19 @@ func New(cfg Config) (*Tracee, error) {
 	// Events chosen by the user
 	for filterScope := range t.config.FilterScopes.Map() {
 		for e := range filterScope.EventsToTrace {
-			t.events[e] = eventConfig{submit: true, emit: true}
+			var submit uint64
+			if _, ok := t.events[e]; ok {
+				submit = t.events[e].submit
+			}
+			submit = 0
```

Comment:
should be in an else branch?

Reply:
Line is entirely omittable, thanks for catching.